src/ATen/native/xpu/mkl/SpectralOps.cpp

```diff
@@ -560,7 +549,7 @@ Tensor _fft_r2c_mkl(
     at::native::_fft_fill_with_conjugate_symmetry_(out, dim);
   }
 
-  return impl::_fft_apply_normalization(out, normalization, input_sizes, dim);
+  return out;
```

Comment:
```
out = std::move(working_tensor);
```
Should not override the original output tensor, as sometimes the output tensor is created by caller.

Reply:
In function `_fft_r2c_mkl`, `Tensor out` is created by itself. Output tensor created by caller is only for `_fft_r2c_mkl_out`.